recipes/rg-etc1/all/CMakeLists.txt

```
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.12)
```

```suggestion
cmake_minimum_required(VERSION 3.4)
```

CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS is only available since cmake 3.4